FFXIVClientStructs/FFXIV/Client/Graphics/Physics/BoneSimulator.cs

```diff
@@ -8,7 +8,21 @@ namespace FFXIVClientStructs.FFXIV.Client.Graphics.Physics;
 [StructLayout(LayoutKind.Explicit, Size = 0x100)]
 public unsafe struct BoneSimulator {
     [FieldOffset(0x00)] public void* vtbl;
-    [FieldOffset(0x10)] public uint PhysicsGroup;
+
+    /// <summary> Non-exhaustive list of physics groups </summary>
+    public enum PhysicsGroup : uint
+    {
+        Clothing = 2,
+        HairA = 3, // Usually the hair's bangs
+        HairB = 4, // Typically the back of the hair
+        HairC = 5, // The sides of the hair
+        HairD = 6, // Extraneous hair bits
+        Chest = 7,
+        Earrings = 8,
+        Ears = 18,
+    }
+
+    [FieldOffset(0x10)] public PhysicsGroup Group;
```

Comment:
Don't know the rules against API changes, is it OK to break? (I'm not sure if there's even any consumers)

Reply:
ABI changes will be auto marked as breaking change due to how C# works, even if there are no consumers for this. This is due to the check system is just comparing it between the old and new CS versions.

Reply:
Ah ok, cool!